x/qgb/types/codec.go

```diff
@@ -23,4 +24,6 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 		&DataCommitment{},
 		&Valset{},
 	)
+
+	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
```

Comment:
[question to understand]
I checked the implementation of this, and it seems to be calling under the hood:

```
registry.RegisterImplementations((*sdk.Msg)(nil),&MsgRegisterEVMAddress{},)
registry.RegisterImplementations((*sdk.Msg)(nil),&MsgRegisterEVMAddressResponse{},)
```

Why do we need to re-register the `MsgRegisterEVMAddress`?
Why do we need to register the `MsgRegisterEVMAddressResponse` even if it's empty?

Reply:
I just noticed this in passing when comparing with the blob module: 

https://github.com/celestiaorg/celestia-app/blob/f78b9459c8f3db1d85c3e579660e425683b14c70/x/blob/types/codec.go#L32

and sdk modules like bank:

https://github.com/celestiaorg/cosmos-sdk/blob/c525ad72ab1449d31115815f5832a68bc79d4ffd/x/bank/types/codec.go#L32

Reply:
I think this just helps for service discovery actually